docs/operator-manual/applicationset/Progressive-Syncs.md

```diff
@@ -105,7 +105,163 @@ spec:
       source:
         repoURL: https://github.com/infra-team/cluster-deployments.git
         targetRevision: HEAD
-        path: guestbook/{{.cluster}}
+  `      path: guestbook/{{.cluster}}
+      destination:
+        server: '{{.url}}'
+        namespace: guestbook
+```
+
+### CRDs
```

Comment:
I think the doc should mention that an "in-line" strategy has priorities over a strategyRef when both are specified. Unless it is possible to add CRD validation to make sure both are not defined, but I am not sure it is possible.

Reply:
Added as documentation